hangups/http_utils.py

```diff
@@ -46,6 +46,9 @@ def fetch(method, url, params=None, headers=None, cookies=None, data=None,
         else:
             error_msg = None
             break
+        finally:
+            if 'res' in locals():
+                res.close()
```

Comment:
> So in aiohttp 2.2.0 it's not possible to avoid the resource warning from aiohttp.request at all? 

Yes. And this is not intended I think.
https://github.com/aio-libs/aiohttp/blob/master/aiohttp/client_reqrep.py#L659
This method is called in ``release``, ``close`` or when ``read`` reached ``EOF``. It just deletes the reference of the session, but they better note who is the owner of the session, similar to the connector. I will file a PR :)